File: Clase6/Python/funciones.py
def saludar(nombre):
    return f"Hola, {nombre}!"

print(saludar("Luis"))


###Listas y areglos:
frutas = ["manzana", "banana", "cereza"]
print(frutas[1])            # banana
frutas.append("naranja")    # agrega
frutas[0] = "pera"           # modifica

a = frutas[0]
a = frutas[1]
for fruta in frutas:
    print(fruta)

##Diccionarios de Python

persona = {
    "nombre": "Luis",
    "edad": 30
}
print(persona["nombre"])
persona["edad"] = 31
persona["ciudad"] = "Guatemala"

for clave, valor in persona.items():
    print(clave, ":", valor)
    

File: Clase6/Java/funciones.java
public class funciones {
    public static String saludar(String nombre) {
        return "Hola, " + nombre + "!";
    }

    public static void main(String[] args) {
        System.out.println(saludar("Luis"));

        String[] frutas = {"manzana", "banana", "cereza"};
        System.out.println(frutas[1]);
        frutas[0] = "pera";

        for (String fruta : frutas) {
            System.out.println(fruta);
        }




    }
    

    
}


